lms/djangoapps/courseware/views/views.py

```diff
@@ -726,7 +726,7 @@ def post(self, request, course_id):
 
 @ensure_csrf_cookie
 @ensure_valid_course_key
-@cache_if_anonymous()
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
```

Comment:
Why do we need to disable cache for the course about page?

Reply:
and do we need this decorator at all? Maybe it's enough to just remove `@cache_if_anonymous()`?